pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -508,10 +508,9 @@ func parseAppInstanceConfig(config *zconfig.EdgeDevConfig,
 		appInstance.FixedResources.VncDisplay = cfgApp.Fixedresources.VncDisplay
 		appInstance.FixedResources.VncPasswd = cfgApp.Fixedresources.VncPasswd
 
-		appInstance.StorageConfigList = make([]types.StorageConfig,
-			len(cfgApp.Drives))
-		parseStorageConfigList(types.AppImgObj, appInstance.StorageConfigList,
-			cfgApp.Drives)
+		appInstance.VolumeRefConfigList = make([]types.VolumeRefConfig,
+			len(cfgApp.VolumeRefs))
+		parseVolumeRefList(appInstance.VolumeRefConfigList, cfgApp.VolumeRefs)
```

Comment:
Presumably the old parseStorageConfigList function can be removed.

Reply:
Not now. Base OS config is using it.